gitlab/v4/cli.py

```diff
@@ -298,13 +304,17 @@ def _populate_sub_parser_by_class(
     if mgr_cls.__name__ in cli.custom_actions:
         name = mgr_cls.__name__
         for action_name in cli.custom_actions[name]:
-            sub_parser_action = sub_parser.add_parser(action_name)
+            sub_parser_action = action_parsers.get(action_name)
+            if sub_parser_action is None:
+                sub_parser_action = sub_parser.add_parser(action_name)
             if mgr_cls._from_parent_attrs:
                 for x in mgr_cls._from_parent_attrs:
                     sub_parser_action.add_argument(
                         f"--{x.replace('_', '-')}", required=True
                     )
-                sub_parser_action.add_argument("--sudo", required=False)
+                sub_parser_action.sub_parser_action.add_argument(
+                    "--sudo", required=False
+                )
```

Comment:
Is this `sub_parser_action.sub_parser_action` intentional here?

Reply:
That was a mistake! Thanks for catching it.